tests/datasource/fluent/test_contexts.py

```diff
@@ -26,6 +26,13 @@
 yaml = YAMLHandler()
 
 
+@pytest.fixture
+def taxi_data_samples_dir() -> pathlib.Path:
+    return pathlib.Path(
+        __file__, "..", "..", "..", "test_sets", "taxi_yellow_tripdata_samples"
+    ).resolve(strict=True)
```

Comment:
Does `resolve` confirm existence? If not, could we do an `exists()` check?

Reply:
With strict=True it does